nilearn/reporting/glm_reporter.py

```diff
@@ -865,14 +885,10 @@ def _stat_map_to_png(
 
     cmap = DEFAULT_DIVERGING_CMAP
 
-    plot_kwargs = {}
+    plot_kwargs = {"threshold": abs(threshold)}
```

Comment:
We even don't need `plot_kwargs`. Isn't it possible to set in function call `threshold=abs(threshold)`?